cinnabar/femap.py

```diff
@@ -25,6 +24,9 @@ def read_csv(filepath: pathlib.Path, units: Optional[openff.units.Quantity] = No
     raw_results = {"Experimental": {}, "Calculated": []}
     expt_block = False
     calc_block = False
+
+    ground = GroundState()
```

Comment:
worth documenting that ground state is assumed to be the same for all inputs for this method?

Reply:
Also is there a unique string we could give it so it hashes differently?